crates/spicepod/src/component/model.rs

```diff
@@ -165,6 +165,23 @@ impl Display for ModelSource {
     }
 }
 
+impl ModelSource {
+    #[must_use]
+    pub fn short_name(&self) -> &'static str {
```

Comment:
This is basically re-implementing the Prefix paradigm that we have for parameters: https://github.com/spiceai/spiceai/blob/trunk/crates/runtime/src/parameters.rs#L230 - instead of inventing an entirely new way to get prefixes for components, we should integrate with the model we already have for datasets.